site/docs/v1/tech/apis/messengers/_generic-response-body.adoc

```diff
@@ -0,0 +1,47 @@
+==== Response Body
+
+[.api]
+[field]#messenger.connectTimeout# [type]#[Integer]#::
+The connect timeout for the HTTP connection, in milliseconds. Value must be greater than `0`.
+
+[field]#messenger.data# [type]#[Object]#::
+An object that can hold any information about the Messenger that should be persisted.
+
+[field]#messenger.debug# [type]#[Boolean]#::
+Determines if debug should be enabled to create an event log to assist in debugging messenger errors.
+
+[field]#messenger.headers# [type]#[Object]#::
+An object that can hold HTTPHeader key and value pairs.
+
+[field]#messenger.httpAuthenticationPassword# [type]#[String]#::
+The basic authentication password to use for requests to the Messenger.
+
+[field]#messenger.httpAuthenticationUsername# [type]#[String]#::
+The basic authentication username to use for requests to the Messenger.
+
+[field]#messenger.id# [type]#[UUID]#::
+The unique Id of the Messenger.
+
+[field]#messenger.insertInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types/#instants[instant] when the Messenger was created.
+
+[field]#messenger.lastUpdateInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types/#instants[instant] when the Messenger was last updated.
+
+[field]#messenger.name# [type]#[String]#::
```

Comment:
Did you intentionally not document `transport`?

Reply:
This is a good catch. I didn't think we used `transport` but alas, I was wrong.